todo_app/data/session_items.py

```diff
@@ -30,7 +30,7 @@ def get_item(id):
     return next((item for item in items if item['id'] == int(id)), None)
 
 
-def add_item(title):
+def add_item(item):
```

Comment:
This change on its own will likely have broken this function, but since we're not using it any more anyway we can just delete this file